packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -713,7 +713,11 @@ It took ${timeInMsToDownloadAllFromCache2}ms to execute download() for all binar
       mockFetch.mockImplementation((url, opts) => {
         opts = opts || {}
         // This makes everything fail with a timeout
-        opts.timeout = 1
+        const controller = new AbortController()
+        setTimeout(() => {
+          controller.abort()
+        }, 1)
+        opts.signal = controller.signal
```

Comment:
**note**

Timeout options doesn't seems to exist anymore. Using signal with abort controller instead.